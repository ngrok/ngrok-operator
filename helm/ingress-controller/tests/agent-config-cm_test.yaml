suite: test agent-config-cm
templates:
- agent-config-cm.yaml
tests:
- it: Should match snapshot
  asserts:
  - matchSnapshot: {}
