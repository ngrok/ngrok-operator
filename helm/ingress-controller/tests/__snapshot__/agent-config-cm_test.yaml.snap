Should match snapshot:
  1: |
    apiVersion: v1
    data:
      LOG: stdout
      METADATA: ""
      REGION: us
      REMOTE_MANAGEMENT: ""
    kind: ConfigMap
    metadata:
      name: RELEASE-NAME-ngrok-ingress-controller-agent-cm
      namespace: NAMESPACE
